tasks/build-examples.js

```diff
@@ -15,6 +14,39 @@ var srcDir = path.join(__dirname, '..', 'examples_src');
 var destDir = path.join(__dirname, '..', 'examples');
 var templatesDir = path.join(__dirname, '..', 'config', 'examples');
 
+
+/**
+ * Cleans the passed JavaScript source from
+ *   * … goog.require … lines
+ *   * … renderer: exampleNS. … lines
+ *   * inline type casts
+ *
+ * @param {String} src The JavaScript source to cleanup.
+ * @returns {String} The cleaned JavaScript source.
+ */
+function cleanupScriptSource(src) {
+  // The regex to replace … goog.require … and … renderer: exampleNS. … lines
+  var lineRegEx = /.*(goog\.require(.*);|.*renderer: exampleNS\..*,?)[\n]*/g;
```

Comment:
Should be defined outside the function to avoid RegEx parsing with every function call.